src/state/application/hooks.tsx

```diff
@@ -111,6 +112,19 @@ export function useNotificationPopup() {
   }, [])
 }
 
+export function useLiquidityV3Popup() {
+  return useCallback(() => {
+    toast.info(<LiquidityV3Popup />, {
+      autoClose: false,
+      icon: false,
+      position: toast.POSITION.BOTTOM_RIGHT,
+      progressStyle: {
+        background: 'hsla(0,0%,100%,.7)',
+      },
+    })
+  }, [])
+}
```

Comment:
I think we can get rid of `useCallback` in this case. This function will only be declared once, so the use of `useCallback` becomes redundant here

Reply:
I just tried this out and it works, pushing the updates in a bit!